langlib/lang.value/src/main/ballerina/value.bal

```diff
@@ -124,6 +171,11 @@ public isolated function isReadOnly(anydata v) returns boolean = @java:Method {
 # The details of the conversion are specified by the ToString abstract operation
 # defined in the Ballerina Language Specification, using the direct style.
 #
+# ```ballerina
+# decimal value = 12.12d;
+# value.toString() ⇒ 12.12
```

Comment:
Shall we add another example with something like a list that has members of few different kinds, which will result in different toString and toBalString representations?